documentation/usage/running-storm-on-parametric-models.md

```diff
@@ -10,108 +10,163 @@ categories: [Use Storm]
 
-The result is an expression over the parameter p.
+The complete output to be expected is:
 
-{% include includes/show_output.html class="closed_form_parametric_models" path="parametric/closed_form.out" %}
+{% include includes/show_output.html class="pla_parametric_models" path="parametric/brp_feasibility_pla.out" %}
 
-## Parameter lifting for region refinement
-For bounded properties we can use *parameter lifting* to partition a given region into safe, unsafe and ambiguous subregions.
+In particular, the output says
 
-### Example 2 (Region refinement for a Bounded retransmission protocol)
-This example is an adaption of the Bounded Retransmission Protocol from the [PRISM website](http://www.prismmodelchecker.org/casestudies/brp.php){:target="_blank"}. Here, we have two channels whose reliabilities are represented by parameters `pL` and `pK`.
+```console
+Result at initial state: 947822915828163/288230376151711744 ( approx. 0.003288421326) at [pK=17/20, pL=17/20].
+```
 
-{% include includes/show_model.html name="parametric version of the Bounded Retransmission Protocol" class="parametric_brp" path="prism/brp.pm" %}
+which means that setting pK and pL to 17/20 yields a Markov chain such that `P<=0.01 [F s=5]` is satisfied.
 
-We want to split our region into those subregions that result in a chance for a successful transmission that is greater than 0.5 and those that do not.
+Instead of giving a bound, pla also support optimization:
 
 ```console
-$ storm-pars --prism brp.pm --prop "P>0.5 [F s=5]" --region "0.1 <= pL <= 0.9, 0.1 <=pK <=0.9" --refine 0.01 10
+$ storm-pars  --mode feasibility --feasibility:method pla --prism brp.pm  --prop 'P=? [F s=5]' --direction min --region "0.1 <= pL <= 0.9, 0.1 <= pK <= 0.9" --guarantee 0.0001 abs
 ```
 
-We use the `--regions` option to set the parameter region.
-The arguments we give to `--refine` are the coverage threshold and the depth limit. The coverage threshold decides how precisely we want to partition our region. The value  0.01 means that Storm will continue to refine until only 1% of the given region remains ambiguous while the rest has been determined to be either safe or unsafe. The depth limit is an optional second argument. If the level at which regions are split reaches this bound, the region is not refined any further, regardless of the coverage achieved so far.
+In this case, we do not give a bound and specify with `--direction min` that we want to minimize.
+Additionally, we give a `--guarantee 0.0001 abs`, specifying that we want to be find a solution which is within 0.0001 of the optimal value, where the distance is measured in absolute terms.
 
-This produces the following output:
+The complete output to be expected is:
 
-{% include includes/show_output.html class="pla_parametric_models" path="parametric/brp_pla.out" %}
+{% include includes/show_output.html class="pla_parametric_models" path="parametric/brp_feasibility_pla_guarantee.out" %}
```

Comment:
By giving all log output the  same CSS class, they are all shown/hidden if clicking on any of them. You can give them separate class names to only enable show/hide per log output.